TestAssets/TestProjects/TestLibrary With Spaces/project.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Honestly, I think we have so many test assets that it is starting to become a maintenance issue. If we need to add another test where the path contains a space, but it also needs something else checked in the project.json, do we need to copy this asset?

Maybe it would be better to create a new TestAsset for `TestLibraryWithXmlDoc`, and then have the test copy the asset into a path with a space in it.  That way the asset isn't so specific as to testing this one thing. It is more reusable.